app/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/SearchGoodsCountryController.scala

```diff
@@ -57,9 +57,11 @@ class SearchGoodsCountryController @Inject()(
         value =>
           repo.upsert(request.declarationJourney.copy(goodsEntries = Seq(
             GoodsEntry(
-              "TODO",
+              Some(CategoryQuantityOfGoods("TODO", "123")),
```

Comment:
I think we should set the CategoryQuantityOfGoods here (which would be non-optional as we have it in hand) and set everything else to 'None'.

We could test the /review-goods page using the test-only page.

Reply:
I think there should actually be a requirement for this page that the CategoryQuantityOfGoods is already set as we need the category for the title/heading "In which country did you buy the x" ... I will pick this up when I revisit this page under it's relevant ticket, didn't want to change too much under this ticket just wanted to address the model as a starter

Reply:
Yeah that's another reason for it to be non-optional here, so that for every subsequent page that needs to display the item category/name has an easier time.

If CategoryQuantityOfGoods is non-optional then we can just check for the existence of the goods entry on subsequent pages and redirect to the /invalid-request page if not set.